GabrielaRamirez/Hackerrank/JavaOutputFormatting.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+
+public class Solution {
+
+    public static void main(String[] args) {
+            Scanner sc=new Scanner(System.in);
+            System.out.println("================================");
```

Comment:
You can set a Constant variable for this String and reuse it
https://deviq.com/antipatterns/magic-strings